sdk/src/driftClient.ts

```diff
@@ -165,7 +166,14 @@ export class DriftClient {
 				: {
 						type: 'websocket',
 				  };
-		this.createUsers(subAccountIds, this.userAccountSubscriptionConfig);
+
+		// load all or some users and add them to the driftClient
+		this.addAllUsers(
```

Comment:
i dont think we want to do any async calls in the constructor, thats why the `createUsers` call didn't call subscribe before

Reply:
maybe we have a new DriftClientConfig Param which is `authoritySubaccountMap` which allows you to specify the authority & subaccount pairs